src/rashdf/geom.py

```diff
@@ -24,32 +27,124 @@ def projection(self) -> Optional[CRS]:
         if type(proj_wkt) == bytes or type(proj_wkt) == np.bytes_:
             proj_wkt = proj_wkt.decode("utf-8")
         return CRS.from_wkt(proj_wkt)
+    
+    def mesh_area_names(self) -> Optional[list]:
```

Comment:
@bates-j this is me being pedantic, but:
1. Would it make sense to return an empty list instead of `None`?
2. Recommend typehint of `Optional[List[str]]`

Reply:
Just want to clarify on 1 above that `None` may in fact be the better answer. Since it can have implications down the road it seemed worth asking. Maybe there are situations where an empty list makes sense and others where `None` makes sense. I guess I lean toward empty list but would be open `None`.

Related discussions:
https://stackoverflow.com/q/5987011
https://stackoverflow.com/q/1969993
https://www.reddit.com/r/learnpython/comments/jdvci3/should_you_return_an_empty_list_on_error_or/

Reply:
Updated to return empty list instead of None for `mesh_area_names`, and to return empty `GeoDataFrame` instead of None where applicable.